### ISSUE TITLE
PostGIS datatype falsely displayed as NULL

### ISSUE THREAD

**Issue by andywicht** [dbeaver/dbeaver#4109]
When it comes to PostGIS Geometry datatypes there is a difference between an empty geometry and a null geometry.

Here is a comparison between the results of pgAdmin 4 and DBeaver 5.1.6
```sql
SELECT gid, st_astext(geom), geom
FROM test
WHERE gid = 137773;
```

Query result pgAdmin (geometries get displayed as HEXEWKB representation)
```
gid    |st_astext        |geom 
-------|-----------------|-----
137773 | "POLYGON EMPTY" | "0103000020E610000000000000"
```

Query result DBeaver
```
gid    |st_astext      |geom 
-------|------------ --|-----
137773 | POLYGON EMPTY | [NULL]    
```

An empty geometry must never be displayed as "[NULL]"

**Comment by andywicht** [comment]
This might be better to reproduce ;)

```sql
SELECT ST_AsText('Polygon EMPTY'::geometry), 'Polygon EMPTY'::geometry;
```

**Comment by serge-rider** [comment]
Well, actually it is a bug (or not?) in Gis processing library. JDBC value can't be extracted with internal error (see below).
I can't fix this particular error. But we can show this error in the grid to make it clear that value isn't null.
This is implemented in #4145
```
org.jkiss.dbeaver.model.exec.DBCException: SQL Error: Error parsing SQL data:java.lang.NegativeArraySizeException
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreGeometryValueHandler.makeGeometryFromString(PostgreGeometryValueHandler.java:90)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreGeometryValueHandler.getValueFromObject(PostgreGeometryValueHandler.java:74)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreGeometryValueHandler.fetchColumnValue(PostgreGeometryValueHandler.java:46)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:49)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetDataReceiver.fetchRow(ResultSetDataReceiver.java:122)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.fetchQueryData(SQLQueryJob.java:726)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.executeStatement(SQLQueryJob.java:462)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.lambda$0(SQLQueryJob.java:389)
	at org.jkiss.dbeaver.model.DBUtils.tryExecuteRecover(DBUtils.java:1598)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:387)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.extractData(SQLQueryJob.java:819)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:2384)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.lambda$0(ResultSetJobDataRead.java:93)
	at org.jkiss.dbeaver.model.DBUtils.tryExecuteRecover(DBUtils.java:1598)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:91)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:95)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:60)
Caused by: java.sql.SQLException: Error parsing SQL data:java.lang.NegativeArraySizeException
	at org.postgis.jts.JtsGeometry.geomFromString(JtsGeometry.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jkiss.utils.BeanUtils.invokeStaticMethod(BeanUtils.java:363)
	at org.jkiss.dbeaver.ext.postgresql.model.data.PostgreGeometryValueHandler.makeGeometryFromString(PostgreGeometryValueHandler.java:86)
	... 16 more
```
